src/eko/io/runcards.py

```diff
@@ -309,6 +312,7 @@ def heavies(pattern: str):
             raise ValueError()
 
         new["xif"] = old["XIF"]
+        new["n3lo_ad_variation"] = old.get("n3lo_ad_variation", (0, 0, 0, 0))
```

Comment:
Why do you need this "old"?

Reply:
here old stands for old format or "old runcard format" i believe...